components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Publisher/src/Deploy/IoTHubPublisherDeployment.cs

```diff
@@ -50,13 +50,33 @@ public sealed class IoTHubPublisherDeployment : IHostProcess {
             await _service.CreateOrUpdateConfigurationAsync(new ConfigurationModel {
                 Id = "__default-opcpublisher-windows",
                 Content = new ConfigurationContentModel {
-                    ModulesContent = CreateLayeredDeployment(false)
+                    ModulesContent = CreateLayeredDeployment(false, false)
                 },
                 SchemaVersion = kDefaultSchemaVersion,
                 TargetCondition = IoTHubEdgeBaseDeployment.TargetCondition +
                     " AND tags.os = 'Windows'",
                 Priority = 1
             }, true);
+            await _service.CreateOrUpdateConfigurationAsync(new ConfigurationModel {
+                Id = "__default-opcpublisher-standalone",
+                Content = new ConfigurationContentModel {
+                    ModulesContent = CreateLayeredDeployment(true, true)
+                },
+                SchemaVersion = kDefaultSchemaVersion,
+                TargetCondition = IoTHubEdgeBaseDeployment.TargetConditionStandalone +
+                    " AND tags.os = 'Linux'",
+                Priority = 1
+            }, true);
+            await _service.CreateOrUpdateConfigurationAsync(new ConfigurationModel {
+                Id = "__default-opcpublisher-windows-standalone",
+                Content = new ConfigurationContentModel {
+                    ModulesContent = CreateLayeredDeployment(false, true)
+                },
+                SchemaVersion = kDefaultSchemaVersion,
+                TargetCondition = IoTHubEdgeBaseDeployment.TargetConditionStandalone +
+                    " AND tags.os = 'Windows'",
+                Priority = 1
+            }, true);
```

Comment:
Can be removed, we don't use Windows hosts for end2end testing

Reply:
Shouldn't we be testing publisher also on Windows ?

Reply:
removed